package/CHANGELOG

```diff
@@ -35,6 +37,7 @@ Changes
   * Removes deprecated ProgressMeter (Issue #2739)
   * Removes deprecated MDAnalysis.units.N_Avogadro (PR #2737)
   * Dropped Python 2 support
+  * Set Python 3.6 as the minimum supported version
```

Comment:
If we are adding this in here, we should probably add in a reference to the original issue for dropping <3.6 #2541

Reply:
Good idea, I didn't remember where we decided this...